server/src/main/java/com/thoughtworks/go/config/update/AgentsEntityConfigUpdateCommand.java

```diff
@@ -102,10 +101,18 @@ public void update(CruiseConfig modifiedConfig) throws Exception {
         validatePresenceOfAgentUuidsInConfig();
         checkIfResourcesAreBeingUpdatedOnElasticAgents();
 
+        validatePresenceOfAgentUuidsInConfig();
+        checkIfResourcesAreBeingUpdatedOnElasticAgents();
         List<AgentConfig> pendingAgents = findPendingAgents();
         validateOperationOnPendingAgents(pendingAgents);
+    }
+
+    @Override
+    public void update(CruiseConfig modifiedConfig) throws Exception {
+        validate();
```

Comment:
This `update` won't be called any more, right? Can we throw an exception from here in that case? It would make it explicit that this will never be invoked.

Reply:
The class `AgentsEntityConfigUpdateCommand` has been removed.